agent/handlers/gql/schema.go

```diff
@@ -404,6 +493,26 @@ func containerNetworksResolver(state dockerstate.TaskEngineState) func(p graphql
 	}
 }
 
+func containerVolumesResolver(p graphql.ResolveParams) (interface{}, error) {
+	dockerContainer, ok := p.Source.(*apicontainer.DockerContainer)
+	if !ok {
+		return nil, nil
+	}
+	return v1.NewVolumesResponse(dockerContainer), nil
+}
+
+func containerHealthResolver(p graphql.ResolveParams) (interface{}, error) {
+	dockerContainer, ok := p.Source.(*apicontainer.DockerContainer)
+	if !ok {
+		return nil, nil
```

Comment:
So in practice, it will never reach "!ok" because the source of the call will always be a `*apicontainer.DockerContainer` pointer. It is only in here to satisfy some unit tests that I created. In production, however, it will never enter this conditional.

Reply:
Thanks for explanation. Please add a comment here.